person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    age >= 18
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
```

Comment:
- [ ] We should use the instance variable `@parent_permission` here instead of `parent_permission`.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```